### DISCUSSION TITLE
Killed - When launch server

### DISCUSSION THREAD

**Discussion by laldevfr** [serverpod/serverpod#539] [Q&A] [answered]
In a DigitalOcean Droplet, i try to launch a serverpod server. 
First time i run with dart bin/main.dart it's ok, but the second time i have only "Killed" message with not more detail. (with or without verbose args)

Dart Version : Dart SDK version: 2.18.6 (stable)  on "linux_x64"
Flutter : Channel stable, 3.3.10, on Ubuntu 22.04.1 LTS 5.15.0-56-generic, locale C.UTF-8 (installed with snap)
Docker : Docker version 20.10.22, build 3a2c30b  (Docker compose work well, i can access to database with a client).

**Comment by yahu1031** [comment, marked as answer]
@laldevfr 
> but the second time I have only "Killed" message with not more detail. (with or without verbose args)

Can you try running it manually with verbose flag. `dart bin/main.dart -v`. So we can see why it was being killed. Can you also please post a screenshot of it if possible?

**Reply by laldevfr** [reply]
I run with verbose flag, but didn't have other message except Killed.
<img width="1089" alt="image" src="https://user-images.githubusercontent.com/89388810/210316279-d18eefc8-986f-4f9a-85df-94052b4ae07f.png">

To notice, in development.yaml file i changed every host with the ip of the server.

**Reply by yahu1031** [reply]
@laldevfr Looks like you don't have enough memory to spawn the dart file/program. Maybe adding a swap will fix it. Can you try it? [Here](https://www.digitalocean.com/community/tutorials/how-to-add-swap-on-ubuntu-14-04) are useful docs for it.

**Reply by yahu1031** [reply]
Something from StackOverflow - [here](https://stackoverflow.com/questions/39642823/digital-ocean-bundle-install-killed), Statamic - [here](https://statamic.com/forum/4857-installation-keeps-getting-killed-on-digital-ocean-droplet#reply-15686)

**Reply by laldevfr** [reply]
Thank a lot, i add a 1G Swap and now i have a readable error, i can keep going ;)
<img width="997" alt="image" src="https://user-images.githubusercontent.com/89388810/210319883-5d7f9435-5d44-45a8-a583-1827941c5e5e.png">
And after reboot everything it's work.

**Reply by yahu1031** [reply]
@laldevfr Awesome, that works. And if you don't mind asking this, Can you please share the steps to how to deploy serverpod in DigitalOcean Droplet. That will be super useful for the community.

**Reply by laldevfr** [reply]
I will write a tutorial after stabilize the deployment. Because i use developpement version with scp of some file ;)

**Reply by yahu1031** [reply]
Sure, Thank you 🌹
